### ISSUE TITLE
Replace wp-cron with system cron job

### ISSUE THREAD

**Issue by jackjamieson2** [jackjamieson2/yarns-microsub-server#24]
@dshanske suggested using a system cron job instead of wp-cron.
https://chat.indieweb.org/dev/2018-09-12/1536774813680400
> 
> GWG
> You should try polling with cron
> GWG
> As opposed to wp cron

> jackjamieson
> Hmmm, any suggestions for references about doing that, or examples? I just did a quick google and it seems like result is referring to wp-cron

> GWG
> Well, look for disable wp cron and run it using a system cron job
> GWG
> But most people won't do that so you still need both

> jackjamieson
> I see - looking this up now

**Comment by jackjamieson2** [comment]
Yeah, I see what you mean. I'd like to prioritize ease of use as much as I can, so this will be a low priority. 

Might think of some other ways to optimize the polling cron. Some brainstorming
- For sites that have low traffic, so might not trigger wp-cron, ensure that requests to the endpoint trigger a poll if it hasn't happened recently
- For sites that have a large number of feeds to poll, maybe split up the polling to make sure it does not monopolize server threads
- Websub could eliminate (or at least deeply reduce) the need for manual polling and would be more efficient overall

**Comment by dshanske** [comment]
This is just a documentation issue on how to do something many people do with their WordPress installations.
